client/js/app.modules/header/header.controller.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * @ngInject
+ * */
+module.exports = function ($scope, $http, GlobalData){
+    this.GlobalData = GlobalData;
```

Comment:
Use lowercase for properties ("globalData"). But its better to show only particular methods on scope (not whole service).